content/about.md

```diff
@@ -1,49 +1,32 @@
----
+te---
 Title: "About"
 ---
-{{< project >}}
+OpenTelemetry provides the libraries, agents, and other components that you need to capture critical telemetry your production services so that you can better observe, manage, and debug them. Specifically, OpenTelemetry captures metrics, distributed traces, and resource metadata from your backend and client applications and then sends this data to backends like Prometheus, Jaeger, Zipkin, [and others](https://opentelemetry.io/registry/?s=exporter) for processing. OpenTelemetry is composed of the following:
```

Comment:
```suggestion
OpenTelemetry provides the libraries, agents, and other components that you need to capture telemetry from your services so that you can better observe, manage, and debug them. Specifically, OpenTelemetry captures metrics, distributed traces, and resource metadata from your backend and client applications and then sends this data to backends like Prometheus, Jaeger, Zipkin, [and others](https://opentelemetry.io/registry/?s=exporter) for processing. OpenTelemetry is composed of the following:
```

Reply:
not only critical telemetry. Also I wouldn't limit the use for production only. Full cycle of developement must be covered

Reply:
/and then send/and send/

Reply:
```suggestion
OpenTelemetry provides the libraries, agents, and other components that you need to capture critical telemetry your production services so that you can better observe, manage, and debug them. Specifically, OpenTelemetry captures metrics, distributed traces, and resource metadata from your backend and client applications and sends this data to backends like Prometheus, Jaeger, Zipkin, and [others](https://opentelemetry.io/registry/?s=exporter) for processing. OpenTelemetry is composed of the following:
```